packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -244,6 +254,7 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
     placeholder: PropTypes.string,
     renderItem: PropTypes.func,
     renderSelectedItem: PropTypes.func,
+    resetHighlightedIndex: PropTypes.number,
```

Comment:
I don't understand why we need this prop when we can control the `highlightedIndex` directly, can you please clarify?

Reply:
we need it so that users don't have to make a whole different `highlightedIndex` controlling logic when they just want for the Dropdown to always open at a specified index. For example Add People picker in WebClient, that opens always with first item highlighted.

Reply:
For me it looks like a case with controlled usage, I think that we should not introduce new props that do exactly the same thing.

/cc @kuzhelov

Reply:
I totally agree. If you look at Downshift's documentation, for controlled props, they have also two helpers, `initial` and `default`. `initial` works similar to our `default` while `default` works similarly to this `reset`. I personally think both of them are useful, along with the actual controlling prop. so if we can converge to a solution, that'd be great.

Reply:
followed up with this https://github.com/stardust-ui/react/issues/970 in which we can discuss more on the subject.